README.md

```diff
@@ -16,20 +16,19 @@ If `-config` is not specified, the default value `helper.conf` is assumed.
+ | `cert_dir`                    | Directory name to store the fetched certificates. This directory must be created previously.                                                                                            | `"certs"`                                                                                                                                                            |
+ | `add_intermediates_to_bundle` | Add intermediate certificates into Bundle file instead of SVID file.                                                                                                                    | `true`                                                                                                                                                               |
+ | `renew_signal`                | The signal that the process to be launched expects to reload the certificates. It is not supported on Windows.                                                                          | `"SIGUSR1"`                                                                                                                                                          |
+ | `svid_file_name`              | File name to be used to store the X.509 SVID public certificate in PEM format.                                                                                                          | `"svid.pem"`                                                                                                                                                         |
+ | `svid_key_file_name`          | File name to be used to store the X.509 SVID private key and public certificate in PEM format.                                                                                          | `"svid_key.pem"`                                                                                                                                                     |
+ | `svid_bundle_file_name`       | File name to be used to store the X.509 SVID Bundle in PEM format.                                                                                                                      | `"svid_bundle.pem"`                                                                                                                                                  |
+ | `jwt_svids`                   | An array with the audience and file name to store the JWT SVIDs. File is Base64-encoded string). | `[{jwt_audience="your-audience", jwt_svid_file_name="jwt_svid.token"}]`                                                                                              |
```

Comment:
@keeganwitt The spacing here seems off, can you take a look?

Reply:
Fixed.